tasks.py

```diff
@@ -17,14 +18,6 @@ def _git_root():
 GIT_ROOT = _git_root()
 PACKAGES_ROOT = GIT_ROOT / "packages"
 
-if platform.system() != "Windows":
```

Comment:
With this removed and the associated usage of it, the tasks no longer work for me. I believe this may be because your terminal automatically activates the venv upon load, but mine does not. As that is a developer setting, I think it might be best to leave this in as it doesn't hurt if it is already active.